BDArmory/Distribution/GameData/BDArmory/Parts/aim-120/amraam.cfg

```diff
@@ -59,6 +59,9 @@ PART
 
 		decoupleSpeed = 5
 
+		engineFailureRate = 0    // Probability the missile engine will fail to start (0-1), evaluated once on missile launch
+		guidanceFailureRate = 0  // Probability the missile guidance will fail (0-1), evaluated every frame after launch
```

Comment:
It might be better for missile builders to have this value as the probability of failure per second instead of per frame and convert it internally as
```csharp
guidanceFailureRatePerFrame = 1f - Mathf.Exp(Mathf.Log(1f - guidanceFailureRate) * Time.fixedDeltaTime);
```